action.yml

```diff
@@ -47,12 +50,10 @@ runs:
     - name: Install lychee
       run: |
         # Cleanup artifacts from previous run in case it crashed
-        rm -rf "lychee-${{ inputs.LYCHEEVERSION }}-x86_64-unknown-linux-gnu.tar.gz" lychee
-        curl -sLO "https://github.com/lycheeverse/lychee/releases/download/${{ inputs.LYCHEEVERSION }}/lychee-${{ inputs.LYCHEEVERSION }}-x86_64-unknown-linux-gnu.tar.gz"
-        tar -xvzf "lychee-${{ inputs.LYCHEEVERSION }}-x86_64-unknown-linux-gnu.tar.gz"
-        rm "lychee-${{ inputs.LYCHEEVERSION }}-x86_64-unknown-linux-gnu.tar.gz"
+        rm -rf lychee* || echo "No leftovers to clean up. Skipping."
+        curl -sL "https://github.com/lycheeverse/lychee/releases/download/${{ inputs.LYCHEEVERSION }}/lychee-x86_64-unknown-linux-gnu.tar.gz" | tar -xvzf -
         install -t "$HOME/.local/bin" -D lychee 
-        rm lychee
+rm -f lychee
```

Comment:
No wildcard here?


```suggestion
rm -f lychee*
```

Reply:
I added the wildcard and fixed the indentation. Whitespace matters when using the suggestion feature 😉

Reply:
That's right